src/main/eo/org/eolang/collections/multimap.eo

```diff
@@ -111,6 +111,81 @@
           a
           a.with x
 
+  [harr arr] > rebuilded
+    mappedi. > wrapped-pairs!
```

Comment:
@EugeneDar Maybe we can rename this object? Something short, like `pairs`, `pairs-list` etc

Reply:
@Graur I agree, I think `sorted-pairs` will be better.